fastestimator/op/numpyop/univariate/reshape.py

```diff
@@ -0,0 +1,24 @@
+from typing import Any, Callable, Dict, Iterable, List, Optional, Tuple, Union
+
+import numpy as np
+
+from fastestimator.op import NumpyOp
+
+
+class Reshape(NumpyOp):
+    """Preprocessing class for reshaping the data
+
+    Args:
+        shape: target shape
+    """
+    def __init__(self,
+                 shape: Union[int, Tuple[int]],
```

Comment:
Union[int, List[int]] since we don't know how big the tuple will be (Tuple typing works like Tuple[int, int, float, etc.] corresponding to exactly what you expect in the tuple